### PR TITLE
Migrated ActionCard.js to TS

### PR DESCRIPTION
**Pull request by hursittarcan**
## Description
- Migrated `ActionCard` component to TypeScript. 

## Related Issue
- Related to [#6392]

### CODE REVIEW

File: src/components/ActionCard.tsx [hunk 1]
```diff
@@ -62,16 +62,27 @@ const Card = styled(Link)`
...
-  to,
-  alt,
-  image,
-  title,
-  description,
-  children,
-  className,
-  isRight,
-  isBottom = true,
+export interface IProps {
```
Comment (pettinarip) [inline src/components/ActionCard.tsx:65]: You made everything optional but there are some props that we do want to make them as required.
How about this?
```ts
export interface IProps {
  to: string
  alt?: string
  image: string
  title: string
  description: string
  className?: string
  isRight?: boolean
  isBottom?: boolean
}
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Awesome @hursittarcan thanks for the PR! 

Please, check the comment I made.

Also, when I ran `yarn type-check` I found an error with the `<ImageWrapper>` component used here. To fix it, you will need to declare the extra props that we are using in this case.

```ts
// on line #20
const ImageWrapper = styled.div<{
  isRight: boolean | undefined
  isBottom: boolean | undefined
}>`
```

After doing that, run again `yarn type-check` to make sure that everything is good.

File: src/components/ActionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                                          to,
+                                          alt,
+                                          image,
+                                          title,
+                                          description,
+                                          children,
+                                          className,
+                                          isRight,
+                                          isBottom = true
+                                      }) => {
```
Comment (pettinarip) [inline src/components/ActionCard.tsx:89]: Hmm weird how this format is in here.. I inspected the last commit and a lot of spaces were added to most of these lines. Do you know why?

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looking great! I noticed a misalignment in the format. Can you fix that?

The format should be covered by `prettier`. Try to do the next:
1. Make a small format change manually in the file
2. Make a new commit
3. `prettier` should format it automatically when you make a new commit

**Comment by hursittarcan** [comment]
Hey @pettinarip, 
I believe the formatting is fixed now. I think the extra spaces were because of my IDE.

**Pull request merged**